config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -224,7 +232,15 @@ expand_tests () {
 # "alpha" features can be breaking across releases and
 # therefore cannot be a required job
 pull_optional() {
-    if [ "$1" == "alpha" ]; then
+    local tests="$1"
+    local kubernetes="$2"
+
+    if [ "$tests" == "alpha" ]; then
+        echo "true"
+    elif [ "$kubernetes" == "1.16.2" ]; then
+        # Testing 1.16 requires release-tools to be updated in all
+        # kubernetes-csi repos. Once that is done, and tests
+        # are passing, this can be removed.
```

Comment:
We are almost done with that. Only https://github.com/kubernetes-csi/node-driver-registrar/pull/53 and https://github.com/kubernetes-csi/external-snapshotter/pull/188 are pending.